convert_all_jupyterlite.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
A comment on why we're doing the conversion with a link to https://github.com/scientific-python/executable-tutorials/issues/59 might be helpful to Future Us.